src/RichContent.js

```diff
@@ -9,6 +9,7 @@ import {
   CANVAS_OBJECT_REFERENCE,
   resourceTypeToHref
 } from "./constants";
+import { getFileResourcePath, mathJaxTypeset } from "./utils";
```

Comment:
I don't see where `getFileResourcePath` is used in this file. Probably needs to be cleaned up.

Reply:
Yes, that's from the other PR in review.